petclinic.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: v1
+kind: List
+items:
+- apiVersion: v1
+  kind: Pod
+  metadata:
+    name: petclinic
+    labels:
+      app: petclinic
+  spec:
+    containers:
+    - name: server
+      image: mariolet/petclinic
```

Comment:
@sleshchenko I would expect this pod to contain mysql only. The spring boot application would run in the mvn-stack tool (with the `start` command). What do you think?

Reply:
It is a recipe for Che 7 workspace that we used for spring boot petclinic sample. 

> I would expect this pod to contain mysql only. The spring boot application would run in the mvn-stack tool (with the start command). What do you think?

It makes sense to me. Let's do this change along with introducing `selector` field in Kubernetes/OpenShift tool. WDYT?